<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBOp_Zeitvertrag</class>
 <widget class="QDialog" name="DBOp_Zeitvertrag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zeitvertrag</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelZeitvertragsID">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ZeitvertragsID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineZeitvertragsID">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelLizenzID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>LizenzID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboLizenzID">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushNeuL">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Neu...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelVertragsbeginn">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vertragsbeginn</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineVertragsbeginn"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelVertragsende">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vertragsende</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineVertragsende"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelRabatt">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Monatsbeitrag</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="doubleMonatsbeitrag">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <double>99999.990000000005239</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEuro">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>â‚¬</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelZahlungsart">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zahlungsart</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboZahlungsart">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Bankeinzug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rechnung</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelBezahltbis">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bezahlt bis</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineBezahltbis">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelVertragsart">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vertragsart</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboVertragsart">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Leasing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maintenance</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>423</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DBOp_Zeitvertrag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DBOp_Zeitvertrag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
